compose.yaml

```diff
@@ -43,6 +43,7 @@ services:
 #      - "SKIP_IP_RANGES=true" # Skip feteching/whitelisting ip ranges from aws and cloudflare, default false
 #      - "LOGROTATE=true" # Enables writing http access logs to /opt/npm/nginx/access.log, stream access logs to /opt/npm/nginx/stream.log and enables daily logrotation, default false
 #      - "LOGROTATIONS=7" # Set how often the access.log should be rotated until it is deleted, default 3
+#      - "LOGROTATEDELAY=25h" # Set the time to wait between running log rotations, default 25 hours. NB this will still only run once a day.
```

Comment:
if you add a new env please add a regex check to the start.sh

Reply:
I'll add it.

Reply:
Done